code/src/bip/datamodel/models.py

```diff
@@ -0,0 +1,221 @@
+import uuid as _uuid
+
+from django.core.validators import MinValueValidator, MaxValueValidator, \
+    RegexValidator
+from django.db import models
+
+from zds_schema.fields import RSINField
+from zds_schema.models import APIMixin
+from zds_schema.validators import (
+    UntilNowValidator, alphanumeric_excluding_diacritic,
+    validate_non_negative_string)
+
+from .constants import GeslachtsAanduiding, NaamgebruikAanduiding, \
+    AdresHerkomst, BurgerlijkeStaat, RedenOpschortingBijhouding, \
+    AanduidingBijzonderNederlandschap
+
+
+class IngeschrevenNatuurlijkPersoon(APIMixin, models.Model):
+    uuid = models.UUIDField(default=_uuid.uuid4)
+
+    burgerservicenummer = models.CharField(
+        null=True, blank=True, max_length=9,
+        validators=[validate_non_negative_string, ],
+        help_text='Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet '
+                  'algemene bepalingen burgerservicenummer.'
+    )
+    a_nummer = models.BigIntegerField(
+        null=True
```

Comment:
wel nullable, maar niet required?

Reply:
Deze begrijp ik niet.

Reply:
Het gaat om de `blank=True` die ik hier verwacht had - in de Django admin bijvoorbeeld kan ik dit veld nu niet leeg laten :)